### PR TITLE
added daisyUI

### PR DESCRIPTION
**Pull request by hayanav**

### CODE REVIEW

**Comment by hayanav** [comment]
adding daisyUI

File: app/(app)/onboarding/onboarding.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+// App.js
+import React from 'react';
```
Comment (ralphr123) [inline app/(app)/onboarding/onboarding.tsx]: No need to import react

File: app/(app)/onboarding/onboarding.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+function App() {
```
Comment (ralphr123) [inline app/(app)/onboarding/onboarding.tsx]: Call the component something like "OnboardingPage" to be clearer

File: app/(app)/onboarding/onboarding.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  return (
+    <div className="container mx-auto px-4">
+      <h1 className="text-6xl text-black">Welcome!</h1>
+      <p className="text-2xl text-purple-600">Let's sign up</p>
+      <div className="text-boxes-container space-y-5">
+        <div className="w-80">
+          <select name="experience" id="experience" className="input input-primary input-bordered w-full">
```
Comment (ralphr123) [inline app/(app)/onboarding/onboarding.tsx]: https://daisyui.com/components/select/

Take a look at the select documentation for daisyui. If you click on the "JSX" table, you can see how they make their select fields:
image

**Pull request closed without merging**